pkg/user/signer.go

```diff
@@ -144,18 +144,39 @@ func (s *Signer) Account(name string) *Account {
 	return s.accounts[name]
```

Comment:
Shouldn't this function also check for new keys added? Why just `AccountByAddress`?